browser-test/src/support/admin_programs.ts

```diff
@@ -1,10 +1,30 @@
 import { Page } from 'playwright'
 
 
-export default class AdminPrograms {
+export class AdminPrograms {
   public page!: Page
 
   constructor(page: Page) {
     this.page = page
   }
+
+  async addProgram(questionNames: string[], programName: string) {
+    await this.page.click('text=Programs');
+    await this.page.click('#new-program-button');
+    await this.page.fill('text=Program Name', programName);
+    await this.page.fill('text=Program Description', "dummy description");
+    await this.page.click('#program-create-button');
+    await this.page.click('text=Edit');
+    await this.page.click('text=Manage Questions');
+    await this.page.fill('text=Block Description', "dummy description");
+    for (const questionName of questionNames) {
+      await this.page.screenshot({ path: `tmp/${questionName}.png`, fullPage: true });
+      await this.page.click(`text=${questionName}`, {force: true});
+    }
+    await this.page.click('#update-block-button');
+    await this.page.click('text=Programs');
+    var pageInnerText = await this.page.textContent('body')
+    await this.page.click('text=Publish');
+    pageInnerText = await this.page.textContent('body')

```

Comment:
Question (blocking): `pageInnerText` isn't used, why is it assigned?

Reply:
Yep! 

```typescript
const el = await this.page.$('text=DRAFT')
assert(el) isn't null
```

https://playwright.dev/docs/api/class-page#pageselector
https://playwright.dev/docs/selectors
https://playwright.dev/docs/assertions

Reply:
ah woops, should be `expect()` rather than `assert`

Reply:
Hm, according to the docs this should work, but in practice the `$` and `click` selectors behave differently.  `$` selectors seem to be just regular js selectors, which doesn't include the `text` stuff.  Used `click` as a little workaround, with comment.